### ISSUE TITLE
Rename "Finished" Release Plan state option to "Complete"

### ISSUE THREAD

**Issue by JonathanCrd** [Azure/azure-sdk-tools#7062]
See https://github.com/Azure/azure-sdk-tools/issues/7051#issuecomment-1745279550 for context.

- [ ] This change has to be done in Dataverse
- [ ] The state change of the Release Plan work item template will need to be changed as well so that we are consistent.
- [ ] Related to https://github.com/Azure/azure-sdk-tools/issues/7079.

**Comment by ladonnaq** [comment]
FYI - That is fine. If we do this then we need to remember to change the state options in the Release Plan work item template.  The state fields are the same as what the Release Planner is using so that we are consistent.

**Comment by JonathanCrd** [comment]
**Change of plans**.
The main reason for this change was to improve the UX of the app, and use the label "Complete" instead of "Finished"

After examining the implementation, I realized that this change would require too much work in the database and the work item for a mere label change. Therefore, I decided to only change the label and leave the back end unchanged (using "Finished" as the completion option).

I have made this change in DEV.
